changelog.asc

```diff
@@ -2,6 +2,40 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.20.0 (TBC)
+
+In 2020, implementing Anki's latest scheduling improvements would have taken years. Today, the same process takes weeks due to the extensive effort of merging Anki's codebase into AnkiDroid. Enjoy your even more efficient reviews!
+
+* Your https://opencollective.com/ankidroid/contribute[donations] paid for the work to make this happen so quickly 🤗
+* *AnkiDroid Updates*
+** Includes Anki 24.11, with FSRS 5.0 (1.4.3)
+*** If you use FSRS, we recommend re-optimizing parameters
+** Forget Cards: Add link to manual
+** Deck Overview: Re-include 'total cards' statistics
+* *Anki 24.11 Features*
+** FSRS 5.0
+*** (experimental) FSRS now schedules same-day reviews if you remove all learning steps
```

Comment:
I'm on the late show, I had to figure out where it was. If I had to, maybe give others a pointer?

```suggestion
*** Haven't tried the new FSRS scheduler? You can enable it in deck options
*** (experimental) FSRS now schedules same-day reviews if you remove all learning steps
```